activiti-api-impl/activiti-api-task-runtime-impl/src/main/java/org/activiti/runtime/api/impl/TaskRuntimeImpl.java

```diff
@@ -420,13 +418,28 @@ public void deleteCandidateGroups(CandidateGroupsPayload candidateGroupsPayload)
         return groupCandidates;
     }
 
+    @Override
+    public List<VariableInstance> variables(GetTaskVariablesPayload getTaskVariablesPayload) {
+        return variableInstanceConverter.from(taskRuntimeHelper.getInternalTaskVariables(getTaskVariablesPayload.getTaskId()).values());
+    }
     
+    @Override
+    public void createVariable(CreateTaskVariablePayload createTaskVariablePayload) {
+    	taskRuntimeHelper.createVariable(false, createTaskVariablePayload);
+    }
     
+    @Override
+    public void updateVariable(UpdateTaskVariablePayload updateTaskVariablePayload) {
+    	taskRuntimeHelper.updateVariable(false, updateTaskVariablePayload);
+    }
+
+    //To do: check if we need this method
```

Comment:
@CTI777 @erdemedeiros as stated in the API PR I think that we should remove this now to avoid future problems. This is here only for legacy reasons.